.scripts/clean-autopr.ts

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+import { exec as execWithCallback } from "child_process";
+import { getAuthenticatedClient } from "./github";
+import { PullRequestsGetAllParams } from "@octokit/rest";
+
+const _repositoryOwner = "Azure";
+
+async function cleanBranches() {
+    const octokit = getAuthenticatedClient();
+    const params: PullRequestsGetAllParams = {
+        owner: _repositoryOwner,
+        repo: "azure-sdk-for-js",
+        state: "open"
+    }
+
+    let pullRequestsResponse = await octokit.pullRequests.getAll(params);
+
+    do {
+        const autoPullRequests = pullRequestsResponse.data.filter(pr => pr.title.startsWith("[AutoPR")).map(pr => pr.head.ref);
+        console.log(JSON.stringify(autoPullRequests, undefined, "  "));
+        console.log(JSON.stringify(autoPullRequests.length, undefined, "  "));
+
+        for (const branch of autoPullRequests) {
+            try {
+                await exec(`git push origin :${branch}`);
```

Comment:
Will this automatically close the PR?

Reply:
Yes, that's what I've been using so far, I just cleaned it enough to check it in :)